code/modules/mob/living/carbon/human/human_damage.dm

```diff
@@ -45,6 +50,12 @@
 	else
 		heal_overall_damage(0, -amount)
 
+/mob/living/carbon/human/adjustBloodLoss(amount)
+	if(amount > 0)
```

Comment:
wtf, no it isn't, also this adjustBloodLoss function is the same as adjustBruteLoss, etc.